test/unit/util.test.ts

```diff
@@ -30,3 +30,29 @@ test('util -> Throttle', async (t) => {
     t.equals(sanitizeString('SELECT * FROM db WHERE something="Poler"'), 'SELECT   FROM db WHERE something  Poler ', 'Escaping query characters');
 
 });
+
+test('Question Mark escape', async(t) => {
```

Comment:
Test name is not correct anymore. The test name should be directly linkable to what is tested, so to the function which is tested.

Reply:
I don't understand here why we have two test for one single function. Does not make sense IMO.

Reply:
Resolved.